src/bika/lims/api/__init__.py

```diff
@@ -817,8 +817,10 @@ def get_url(brain_or_object):
     :returns: Absolute URL
     :rtype: string
     """
-    if is_brain(brain_or_object) and base_hasattr(brain_or_object, "getURL"):
-        return brain_or_object.getURL()
+    if is_brain(brain_or_object):
+        request = get_request()
+        path = get_path(brain_or_object)
+        return request.physicalPathToURL(path, relative=0)
```

Comment:
Good point, better rely on `get_path` and `physicalPathToURL` instead of waking up the object unnecessarily. Note to @xispa: remove existing calls to brain's `getURL` column (still present in listings like `AnalysesView` or `SamplesView`).